src/storage/tuple.cpp

```diff
@@ -230,14 +230,14 @@ void Tuple::DeserializeFrom(UNUSED_ATTRIBUTE SerializeInput &input,
    * out of Tuple. The memory allocation will be performed when serializing
    * to tuple storage.
    */
-  /*const bool is_inlined = tuple_schema->IsInlined(column_itr);
+  /*const bool is_inlined = tuple_schema_->IsInlined(column_itr);
```

Comment:
Hmm... looks like it's some ancient code from last year that used to load tuples from log files or maybe csv/tsv files. I think may be I can revisit it when reviving the logging module.